sdk/storage/azure-storage-queue/tests/test_queue_service_stats.py

```diff
@@ -39,27 +41,29 @@ def _assert_stats_unavailable(self, stats):
     def override_response_body_with_unavailable_status(response):
         response.http_response.text = lambda: SERVICE_UNAVAILABLE_RESP_BODY
 
+    @staticmethod
+    def override_response_body_with_live_status(response):
+        response.http_response.text = lambda: SERVICE_LIVE_RESP_BODY
```

Comment:
Why do we have to overwrite the live status response? Does this not work live?

Reply:
Creating a secondary location and making it available is either taking a lot of time or doesn't work live.